README.md

```diff
@@ -1,4 +1,113 @@
+</p>
+
+<p align=center>
+<a href="https://pypi.org/project/finetuner/"><img src="https://img.shields.io/badge/Python-3.9%2B-blue alt="Python 3.9" title="Finetuner supports Python 3.9 and above"></a>
+<a href="https://slack.jina.ai"><img src="https://img.shields.io/badge/Slack-2.2k%2B-blueviolet?logo=slack&amp;logoColor=white"></a>
+</p>
+
+<!-- start elevator-pitch -->
+
+Fine-tuning deep neural networks (DNNs) significantly improves performance on domain specific neural search tasks.
+However, fine-tuning for neural search is not trivial, as it requires a combination of expertise in ML and Information Retrieval.
+Finetuner makes finetuning simple and fast by handling all related complexity and infrastructure in the cloud. With Finetuner, you can easily make models more perfomant and production ready.
+
+📈**Performance boost**: Finetuner significantly increases the performance of pretrained models on domain specific neural search applications.
+
+🔱 **Simple yet powerful**: Interacting with Finetuner is simple and seamless, and also supports rich features such as
+siamese/triplet loss, metric learning, self-supervised pretraining, layer pruning, weights freezing, dimensionality reduction, and much more.
+
+☁ **Finetune in the cloud**: Finetuner runs your fine-tuning jobs on [Jina Cloud](https://github.com/jina-ai/jcloud). You never have to worry about provisioning (cloud) resources! Finetuner handles all related complexity and infrastructure.
+
+<!-- end elevator-pitch -->
+
+## What is the purpose of Finetuner?
+
+Finetuner enables performance gains on domain specific neural search tasks by allowing you to fine-tune models in the cloud. We have conducted experiments on three neural search tasks in different domains to illustrate these performance improvements.
+
+| Task              | ndcg score (pre-trained)  | ndcg score (fine-tuned)   | Performance gains (∂) | Time cost |
+|-------------------|--------------------------:|--------------------------:|----------------------:|----------:|
+| text-to-text      |                           |                           |                       |           |
+| image-to-image    |                           |                           |                       |           |
+| text-to-image     |                           |                           |                       |           |
+
+Finetuner also aims to make fine-tuning simple and fast. When interacting with Finetuner, the API takes care of all your fine-tuning jobs in the cloud. This only requires a few lines of code from you, as demonstrated in section 
+
+## How does it work?
+
+<img src="https://github.com/jina-ai/finetuner/blob/docs-update-readme/docs/_static/finetuner-client-journey.svg?raw=true" title="Finetuner Client user journey.">
+
+
+## Install
+
+Requires Python 3.7+ installed on Linux/MacOS.
+
+```bash
+pip install -U finetuner-client
+```
```

Comment:
just a remark, we need to manage conda later

Reply:
added this to our ToDo document.